src/services/companyService.ts

```diff
@@ -10,6 +12,13 @@ export const getCompanies = (
   containerName: NonEmptyString,
   blobName: NonEmptyString
 ) =>
-  getBlobData(blobServiceClient, containerName, blobName, UsersCompanies)
-    .map(_ => _.find(elem => elem.fiscalCode === fiscalCode))
-    .map(result => fromNullable(result).map(_ => _.companies));
+  pipe(
+    getBlobData(blobServiceClient, containerName, blobName, UsersCompanies),
+    TE.map(_ => _.find(elem => elem.fiscalCode === fiscalCode)),
```

Comment:
Wouldn't it be better to avoid this kind of "_" usage? _ is a blob data containing what? Maybe a more significative name would help to understand "what you search and where you search it".